cypress/e2e/awx/resources/inventoriesConstructed.cy.ts

```diff
@@ -0,0 +1,167 @@
+import { randomString } from '../../../../framework/utils/random-string';
+import { InstanceGroup } from '../../../../frontend/awx/interfaces/InstanceGroup';
+import { Inventory } from '../../../../frontend/awx/interfaces/Inventory';
+import { Organization } from '../../../../frontend/awx/interfaces/Organization';
+import { awxAPI } from '../../../support/formatApiPathForAwx';
+
+describe('Constructed Inventories CRUD Tests', () => {
+  let organization: Organization;
+  let inventoriesList: Inventory[] = [];
+  let invNames: string[] = [];
+  let constructedInv: Inventory;
+  let instanceGroup: InstanceGroup;
+  const invToDelete: Inventory[] = [];
+  const constrInvToDelete: Inventory[] = [];
+
+  before(() => {
+    cy.awxLogin();
+
+    const orgName = 'E2E Organization Constructed Inventory tests ' + randomString(4);
```

Comment:
IMO, the name here is too long as with the random string it will be even lengthier. suggestion to please change it to `E2E Org`